bookfoot.php

```diff
@@ -55,7 +55,7 @@ function onSelect() {
 ?>
 <hr/>
 <p>
-If you find a mistake in this book, feel free to send me a fix using
+Si encuentras un error en este libro, siéntete libre de enviarme una solución usando
 <a href="https://github.com/csev/py4e/tree/master/book3" target="_blank">Github</a>.
```

Comment:
I think it would be better to change the URL to the csve-es repository.
https://github.com/csev-es/py4e/tree/master/book3

Reply:
efectuada la corrección de la url.